x-pack/plugins/index_management/common/types/data_streams.ts

```diff
@@ -59,7 +59,9 @@ export interface DataStream {
   _meta?: Metadata;
   privileges: Privileges;
   hidden: boolean;
-  lifecycle?: IndicesDataLifecycleWithRollover;
+  lifecycle?: IndicesDataLifecycleWithRollover & {
```

Comment:
ES types havent been updated yet to include the `enabled` prop, so I'm manually adding it to our type definition for now